docs/src/development/ssh/_index.md

```diff
@@ -0,0 +1,167 @@
+
+If you don't find an existing key, [generate new keys](#generate-new-keys).
+
+### Add an SSH key to your Platform account
+
+Once you have the location of your public key, add it to your Platform.sh account.
+
+If you're logged in using the [Platform.sh CLI](#authenticate-with-the-platformsh-cli), in a terminal run the following command (replacing `PATH_TO_YOUR_KEY` with the location of your public key):
+
+```bash
+platform ssh-key:add 'PATH_TO_YOUR_KEY`
+```
+
+You can also add it in the management console, similar to this [video](https://docs.platform.sh/videos/management-console/add-ssh-mc.mp4).
+
+Now you are ready to use the key to [connect to an environment](#connect-to-your-server-with-ssh).
+
+### Generate new keys
+
+If you're logged in using the [Platform.sh CLI](#authenticate-with-the-platformsh-cli), generate a key and have it added to your Platform.sh account automatically.
+
+1. In a terminal, run `platform ssh-key:add`.
+1. If necessary, log in to a browser.
+1. Press `Y` and `enter` to create a new SSH key.
+1. Copy the location of the generated key.
+1. Run the following commands (replacing `PATH_TO_YOUR_KEY` with the location you copied):
+
+   ```bash
+   $ eval $(ssh-agent)
+   $ ssh-add 'PATH_TO_YOUR_KEY`
+   ```
+
+To generate a key otherwise, GitHub has a good [walk-through for creating SSH key pairs](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) on various operating systems. Then you need to [add it to your Platform.sh account](#add-an-ssh-key-to-your-platform-account).
+
+Now you have a public and a private key and the public key is added to your account. You are ready to use the keys to [connect to an environment](#connect-to-your-server-with-ssh).
+
+### Forwarding keys by default
+
+It may be helpful to set your SSH client to always forward keys to Platform.sh servers, which can simplify other SSH or rsync commands. To do so, include a block in your local `~/.ssh/config` file like so:
+
+```text
+Host *.us.platform.sh
+       ForwardAgent yes
+Host *.eu.platform.sh
+       ForwardAgent yes
+```
+
+Include one `Host` entry for each Platform.sh region you want to connect to, such as `us-2` or `eu-4`. (You can include other configuration as desired.)
+
+## Connect to your server with SSH
+
+To access an environment via the CLI:
+
+1. In a terminal, run `platform ssh`.
+1. (If not currently in a project directory) enter the number of the project you want to access.
+1. (If there are multiple environments) enter the ID of the environment you want to access.
+1. (If there are multiple apps) enter the number of the app you want to access.
+
+To connect using SSH keys, find the details in the management console:
+
+1. Open the [Platform.sh console](https://console.platform.sh/).
+1. Select a project.
+1. In the **Environment** dropdown, select the environment you want to access.
+1. Click the **SSH** dropdown.
+1. Copy the ssh command for the app you want.
```

Comment:
```suggestion
1. Copy the ssh command for the environment you want.
```

Reply:
About the change app -> environment, it's not the environment if it's in a mult-app setup. That's why I went with `app`.

Reply:
oh, right - if that's the case it could be app, environment or host (in Dedicated Gen 1 and 2, they connect directly to the host) but I guess that belongs in the "Dedicated" sections ? Either way, I'm happy to disregard

Reply:
Is it OK now as `for where you want access`, @MariadeAnton ?